modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkByScrollTask.java

```diff
@@ -208,7 +215,11 @@ public XContentBuilder innerXContent(XContentBuilder builder, Params params)
             builder.field("batches", batches);
             builder.field("version_conflicts", versionConflicts);
             builder.field("noops", noops);
-            builder.field("retries", retries);
+            builder.startObject("retries"); {
```

Comment:
This is a breaking change now so this PR needs to be marked as "breaking" and a note added to the migration guide